net/init-metrics.sh

```diff
@@ -83,11 +83,14 @@ else
   query "CREATE DATABASE \"$netBasename\""
   query "ALTER RETENTION POLICY autogen ON \"$netBasename\" DURATION 7d"
   query "GRANT READ ON \"$netBasename\" TO \"ro\""
-  query "GRANT WRITE ON \"$netBasename\" TO \"scratch_writer\""
+  query "GRANT WRITE ON \"$netBasename\" TO \"${username}\""
 
-  SOLANA_METRICS_CONFIG="host=$host,db=$netBasename,u=scratch_writer,p=topsecret"
+  SOLANA_METRICS_CONFIG="host=$host,db=$netBasename,u=${username},p=${password}"
+  set +x
```

Comment:
Why `set +x` here?

Reply:
So it would not print the command itself and only output the stdout.

Reply:
Ah, the way `-` and `+` work for `set` is always confusing to me %)

`+x` disables the command printing.

I think the issue here is that `query` enables it, but forgets to disable:

```bash
  query() {
    echo "$*"
    set -x
    curl -XPOST \
      "$host/query?u=${username}&p=${password}" \
      --data-urlencode "q=$*"
  }
```

Instead of adding `set +x` here, I think, it would be better to add it at the end of `query()`.
Or remove `set -x` from `query()`.
Is it really convenient that it is printing all those query commands?
It looks more like debugging leftovers, than an actually useful functionality.